recipes/default.rb

```diff
@@ -21,6 +21,11 @@
 include_recipe 'hadoop::_hadoop_checkconfig'
 include_recipe 'hadoop::_compression_libs'
 
+# Load helpers
+Chef::Recipe.send(:include, Hadoop::Helpers)
```

Comment:
It probably makes sense to do the Recipe include in the helper module itself.  All the service-specific recipes that include ``default`` anyway are just repeating these include statements.  Putting it in the module would load it once and make it available to all recipes.